core/lib/dal/migrations/20241209150000_create_via_votes.up.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS via_votable_transactions (
+    l1_batch_number BIGINT NOT NULL REFERENCES l1_batches (number) ON DELETE CASCADE,
+    tx_id BYTEA,
+    is_finalized BOOLEAN NOT NULL DEFAULT FALSE,
+    created_at TIMESTAMP NOT NULL DEFAULT NOW(),
+    updated_at TIMESTAMP NOT NULL DEFAULT NOW(),
+    PRIMARY KEY (l1_batch_number, tx_id)
```

Comment:
We should never have a block_number with 2 transaction validation, right? If yes, I think we should keep only the l1_batch_number as PK, because this can lead to double transactions for same block_number.

Reply:
I put this here to just check that everything related to proof inscription works fine. let's keep it like this for devnet-2 also